main.go

```diff
@@ -45,8 +45,10 @@ func main() {
 	}
 
 	if *token == "" {
-		level.Error(logger).Log("err", "-token is required")
-		os.Exit(1)
+		if *token = os.Getenv("FASTLY_API_TOKEN"); *token == "" {
+			level.Error(logger).Log("err", "-token or FASTLY_API_TOKEN is required")
```

Comment:
Please also amend the help text for the -token flag to e.g.

```go
token = fs.String("token", "", "Fastly API token (required; also via FASTLY_API_TOKEN)")
```

Reply:
Addressed in https://github.com/peterbourgon/fastly-exporter/pull/18/commits/29b472ab3c90bc5d05b87a269f7b8d25338856e1